app/assets/javascripts/models/beta/story.js

```diff
@@ -123,6 +128,15 @@ export const updateStory = (story, newAttributes) => ({
   ...newAttributes
 });
 
+export const search = async (keyWord, projectId) => {
+  const { data } = await httpService
```

Comment:
Add a `try/catch` to this request if you don't have yet on search method call.

Reply:
I think we should keep it at call site, yes.